dialogue-apache-hc5-client/src/main/metrics/dialogue-apache-client-metrics.yml

```diff
@@ -40,6 +40,11 @@ namespaces:
         tags: [client-name, client-type]
         docs: Reports the time spent creating a new connection. This includes both connecting the socket and the full TLS handshake.
 
+      connection.route.error:
+        type: meter
+        tags: [client-name]
```

Comment:
let's tag with `cause` and use the throwable simple name

Reply:
I also added `client-type` to match the other connection metrics